fs-storage/src/bin/cli.rs

```diff
@@ -0,0 +1,64 @@
+use std::collections::BTreeMap;
+use std::env;
+use std::path::Path;
+
+use fs_storage::file_storage::FileStorage;
+
+fn main() {
+    let args: Vec<String> = env::args().collect();
+    if args.len() < 3 {
+        println!("Usage:");
+        println!("    cargo run -- write <path> <key1:value1,key2:value2,...>");
```

Comment:
Using `cargo run -- write <path> <key1:value1,key2:value2,...>` may become challenging if we need to pass several key-value pairs. 
Maybe providing them in JSON format would be a cleaner approach. What do you think?

Reply:
It can also be easier to parse and use than splitting each time

Reply:
@tareknaser, yes, it will be better. working on it